python/bids_import.py

```diff
@@ -271,8 +271,8 @@ def create_loris_bids_directory(bids_reader, data_dir, verbose):
 
     # determine the root directory of the LORIS BIDS and create it if does
     # not exist
-    name = re.sub("[^0-9a-zA-Z]+", "_", bids_reader.dataset_name) # get name of the dataset
-    version = re.sub("[^0-9a-zA-Z\.]+", "_", bids_reader.bids_version) # get BIDSVersion of the dataset
+    name = re.sub("[^0-9a-zA-Z]+", "_", bids_reader.dataset_name)        # get name of the dataset
+    version = re.sub("[^0-9a-zA-Z\\.]+", "_", bids_reader.bids_version)  # get BIDSVersion of the dataset
```

Comment:
Est0ce que ça fonctionne encore si on escape le slash?

Reply:
The old version triggered a [Flake W605](https://www.flake8rules.com/rules/W605.html) on my end, but that was not a proper solution. I updated it, it should be good now with `r"regex"` string pattern.